src/position-value.js

```diff
@@ -0,0 +1,45 @@
+// Copyright 2015 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//     You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//     See the License for the specific language governing permissions and
+// limitations under the License.
+
+(function(shared, scope, testing) {
+
+  /**
+   * PositionValue(xPos, yPos)
+   */
+  function PositionValue(xPos, yPos) {
+    //Check that method parameters are of type LengthValue
```

Comment:
Can delete this comment, it's not necessary as it's clear that you're checking the types of xPos and yPos below.